mods/cnc/tilesets/desert.yaml

```diff
@@ -264,9 +264,9 @@ Templates:
 		Size: 3,2
 		Category: Cliffs
 		Tiles:
-			0: Clear
+			0: Rock
```

Comment:
Are you sure about this change?
![tdcliff](https://cloud.githubusercontent.com/assets/7704140/12011086/f17a6872-acc0-11e5-8f50-bb535bff7ca9.png)
looks like it should be passable.